server/env.ts

```diff
@@ -11,6 +11,8 @@ const env = cleanEnv(process.env, {
   DB_USER: str(),
   DB_PASSWORD: str(),
   DB_SSL: bool({ default: false }),
+  DB_POOL_MIN: num({ default: 1 }),
+  DB_POOL_MAX: num({ default: 2 }),
```

Comment:
Can you update it to the actual default amounts which is min 2 and max 10?

Reply:
Done.